### ISSUE TITLE
Invalid Syntax::: SyntaxError: Missing parentheses in call to 'print'

### ISSUE THREAD

**Issue by NovaCygni** [derv82/wifite#91]
[+] downloading update...
Archive:  /tmp/wifiteEY9J9z/wifite-mod-pixiewps-master.zip
cb8a19e3e1a6d79f0ca50aea5fb7be6400dd903e
   creating: /tmp/wifiteEY9J9z/wifite-mod-pixiewps-master/
  inflating: /tmp/wifiteEY9J9z/wifite-mod-pixiewps-master/LICENSE
  inflating: /tmp/wifiteEY9J9z/wifite-mod-pixiewps-master/README.md
  inflating: /tmp/wifiteEY9J9z/wifite-mod-pixiewps-master/wifite-ng
 [+] Updated
 [+] quitting

[antisec@Firewall Desktop]$ sudo wifite --help
  File "/usr/bin/wifite", line 180
    print R+' [!]'+O+' ERROR:'+G+' wifite'+O+' must be run as '+R+'root'+W
          ^
SyntaxError: Missing parentheses in call to 'print'

**Comment by 13337** [comment]
@NovaCygni My initial thought is "try running it with python 2.7 instead of python 3"

**Comment by NovaCygni** [comment]
If the "Update" process changes the default Python the program is calling id suggest putting the correct call for Python at the top, as it stands installing wifite from the AUR, then performing the in program "update" results in a wifite that can no longer simply be called with "wifite" in the console...

**Comment by 13337** [comment]
a couple notes:

I'm not one of the devs. I just happen to be familiar with wifite, was passing through, say that there was an unanswered issue that I could provide a little help with. Furthermore, it doesn't look like wifite is being all that actively developed, juding by the backlog of pull requests going all the way back to sometime in 2014. If the case were different, I'd absolutely love to add one character to the end of the shebang and submit a pull request just so I could say I'd contributed to an open source project that gets such wide use from all the dipshit skiddies in the world who can't be bothered to throw a fucking '2' on the end of line fucking 1 of a python script. And on that note, if you can use Arch, and install anything from the AUR, you don't need the devs of this software to make the modication. FURTHERMORE, SOME PEOPLE AREN'T USING PYTHON. I'm absolutely positive that not every copy of wifite running right now is running on a system where /usr/bin/python isn't python3, and vice versa.

TL;DR, sorry, not sorry, if you want to use hacking tools, maybe you shouldn't be afraid to 'sudo nano /usr/bin/wifite' and modify the shebang yourself. From my viewpoint as someone who isn't a dev on this project, the laziest person involved in this issue is the one who can't be bothered to do some unbelievably minor modification to code running on their machine.

**Comment by NovaCygni** [comment]
Actually ive no issue with making it work myself I was putting a bug report in so they could be aware of a slight issue, also the AUR version the same as the one in Blackarchs Repos is Revision 87, well out of date ;o Good bug reporting from users enables Devs who are active to solve these issues, I have my own HackTool on Github, and due to lack of free time to test everything myself BugReports are a big help.

I was actually considering Porting wifite to Python3 and maintaining a branch myself due to the apparent abandoned status of the project, though at the moment im porting CloudBuster over and updating that as again, it appears abandoned :) 

Think you assumed I was being a arsehole I was merely pointing out a bug in the program.
